contracts/PatriciaTree/PatriciaTreeBase.sol

```diff
@@ -40,7 +40,7 @@ contract PatriciaTreeBase is PatriciaTreeProofs {
       Data.Label memory prefix;
       Data.Label memory suffix;
       (prefix, suffix) = k.splitCommonPrefix(e.label);
-      assert(prefix.length == e.label.length); // I.e. never an unseen branch
+      assert(prefix.length == e.label.length); // I.e. never an unseen branch ignore-swc-110
```

Comment:
I don't understand the comment here. Or is it just me struggling to process double negatives?